src/lib/components/Upload_ReactComponent.react.js

```diff
@@ -50,15 +49,15 @@ export default class Upload_ReactComponent extends Component {
 
         // Full list of options here
         // https://github.com/flowjs/flow.js#configuration 
-        const flowComponent = new Flow({
+        const flow = new Flow({
             //  The API endpoint
             target: this.props.service,
             // Additional data for the requests
             query: { upload_id: this.props.upload_id },
             // Chunk size in bytes.
             chunkSize: this.props.chunkSize,
             // Number of simulateneous uploads
-            simultaneousUploads: this.props.simultaneousUploads,
+            simultaneuosUploads: this.props.simultaneuosUploads,

```

Comment:
Was there a typo previously..? The "simultaneous" would be correct English. Why the change?